Tasks/Sergey Yashchanka/Task_3/count_letters.py

```diff
@@ -0,0 +1,47 @@
+# Задание:
+
+# 1. Написать программу для поиска наиболее часто встречающихся элементов в строке.
+# Пользователь вводит строку, и количество элементов, которые он хотел бы увидеть в ответе.
+#
+# Пример:
+# Строка:lkseropewdssafsdfafkpwe
+# Элементы:3
+# 's' встречается 4 раза
+# 'e' встречается 3 раза
+# 'f' встречается 3 раза
+
+# ------------------------------------------------------------------------------------------------------------------ #
+
+# Решение:
+
+string = input("please input your text | ")
+elements = int(input("please input amount of elements | "))
+count_dict = {}
+
+for c in string.replace(" ", ""):  # get a dictionary with the number of repetitions of each element
+    if c in count_dict:
+        count_dict[c] += 1
+    else:
+        count_dict[c] = 1
+
+sorted_values = sorted(count_dict.values())[::-1]  # sort elements by value
+sorted_dict = {}
+for i in sorted_values:  # get a sorting dictionary  with help 'sorted_values'
+    for key in count_dict.keys():
+        if count_dict[key] == i:
+            sorted_dict[key] = count_dict[key]
```

Comment:
and this part is not optimal at all, you can try to add some logging or debug it and see for yourself how much meaningful assigning it does